docs/data/install-xrootd.md

```diff
@@ -287,13 +287,32 @@ fermicloud054.fnal.gov complete inventory as of Tue Apr 12 07:38:29 2011 /data/x
 
 **Note**: In this example, `fermicloud53.fnal.gov` is a redirector node and `fermicloud054.fnal.gov` is a data node.
 
+### (Optional) Enabling a FUSE mount
```

Comment:
If this is just for GridFTP + XRootD configurations, can we just add this as a "Verification" section of the "Install GridFTP doc"?

Reply:
That's the interesting part - it's not needed for GridFTP + XRootD.  One could vaguely use it to help debug ... maybe?

Reply:
Well that makes the first sentence misleading. In that case, I vote that we just remove this section.

Reply:
Nah, it's a useful section -- POSIX mounts are generally beloved by users.  Just not related to GridFTP.

Removing the sentence.